src/components/MermaidDash/MermaidDash.jsx

```diff
@@ -214,7 +229,7 @@ const MermaidDash = () => {
     )
   }
 
-  const renderMap = () => (
+  const map = (
```

Comment:
Nested react components can cause performance issues. I converted a couple to simpler JSX elements. In my next PR I will be auditing the code for more nested components and converting them to JSX if possible. (also this can cause bugs, not just performance issues)